pkg/crc/machine/machine.go

```diff
@@ -53,14 +54,26 @@ func Start(startConfig StartConfig) (StartResult, error) {
 		Memory:     startConfig.Memory,
 	}
 
-	logging.InfoF("Extracting the Bundle tarball ...")
+	logging.InfoF("Extracting the %s Bundle tarball ...", filepath.Base(machineConfig.BundlePath))
 	crcBundleMetadata, extractedPath, err := bundle.GetCrcBundleInfo(machineConfig)
 	if err != nil {
 		logging.ErrorF("Error to get bundle Metadata %v", err)
 		result.Error = err.Error()
 		return *result, err
 	}
 
+	// Verify, user is using latest bundle with released version of crc.
+	if constants.DefaultHostname != crcBundleMetadata.Nodes[0].Hostname {
+		hypervisor := machineConfig.VMDriver
+		if hypervisor == "virtualbox" {
+			hypervisor = "vbox"
```

Comment:
why is this necessary?

Reply:
@gbraad because our bundle contain `vbox` instead `virtualbox` in the tarball, we can either change it there, make part of snc script or here.

Reply:
this is not the place to change/override it... this feels like a workaround value.

@code-ready/crc-devel WDYT?

Reply:
@gbraad yes this is workaround, I will create an issue on snc side so we can generated the tarball with hypervisor next time, Will put Note here meanwhile.

Reply:
@gbraad https://github.com/code-ready/snc/issues/56 here

Reply:
If this is approved and merged, will we eventually 'remove' this with the next release?

Reply:
@gbraad yes we will remove that part from next release.